generator/sig_readme.tmpl

```diff
@@ -18,6 +22,9 @@ Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 ## Contact
 * [Slack](https://kubernetes.slack.com/messages/{{.Contact.Slack}})
 * [Mailing list]({{.Contact.MailingList}})
+{{- if .Label }}
+* [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2F{{.Label}})
```

Comment:
sorry for delay. but I think people most possibly want to see a sig's open issues/prs in k/k, not in community. @cblecker WDYT?

Reply:
Wouldn't be bad to add both. I was more thinking about looking at open design proposals for the sig.

Reply:
+1 for adding both :)